contracts/StakeLockContract.sol

```diff
@@ -0,0 +1,109 @@
+pragma solidity >=0.4.22;
+
+contract StakeLockContract {
+    /**
+     * @dev Error messages for require statements
+     */
+    string internal constant ALREADY_LOCKED = 'Tokens already locked';
+    string internal constant NOT_LOCKED = 'No tokens locked';
+    string internal constant AMOUNT_ZERO = 'Amount can not be 0';
+
+    uint256 internal constant LOCK_PERIOD_IN_EPOCHS = 3;
+
+    uint256 internal numBlocksPerEpoch = 5;
```

Comment:
this is for testing only, please add a comment.

Reply:
Done.